src/main/java/org/rpgl/subevent/CriticalHitDamageCollection.java

```diff
@@ -20,9 +20,10 @@ public CriticalHitDamageCollection() {
     }
 
     @Override
+    @SuppressWarnings("all")
```

Comment:
Why is this here?

Reply:
to avoid a warning about not calling super.clone(). More specific warning suppression seems not to work.